src/redux/redux-store.js

```diff
@@ -0,0 +1,6 @@
+import * as Redux from "redux";
+import appReducer from "./appReduxer";
+import {applyMiddleware} from "redux";
+import thunk from "redux-thunk";
+
+export default Redux.createStore(appReducer, applyMiddleware(thunk))
```

Comment:
Create ``configureStore`` function that returns state. 

**Note**: cuz it helps to perform isolated store testing. Even if you don't have tests, please do this. And hearts of developers begin to thaw when you will be doing this 🥶

Reply:
```
import appReducer from "./appReduxer";
import { configureStore } from "@reduxjs/toolkit";

export default configureStore({
	reducer: appReducer
})
```